app/models/grouping.rb

```diff
@@ -229,5 +229,7 @@ def accept_tracker_story
 
     story.current_state = "accepted"
     story.save
+
+    rescue
```

Comment:
This might bite us in the future, but we have a plan to use current user's tracker API tocker in any case, so... LGTM!